### PR TITLE
chore: adds setup to readme

### PR DESCRIPTION
**Pull request by yowainwright**
## Description

This PR adds initial setup steps for JSPM.ORG.

- It adds a `.nvmrc` to specify a node version.
- It adds a new symlink when outputting a new version of docs

### CODE REVIEW

File: public_html/feed.json [hunk 1]
```diff
@@ -10,39 +10,39 @@
       "title": "Deprecating jspm.dev",
       "content_html": "<p>With import maps fully supported in browsers, the JSPM project is moving to only support import map CDN workflows via the ga.jspm.io CDN, while jspm.dev is being deprecated over a two year timeframe.</p>",
       "date_published": "2024-01-13T20:21:45.600Z",
-      "date_modified": "2024-01-14T18:26:30.151Z"
+      "date_modified": "2024-03-25T03:35:50.578Z"
```
Comment (guybedford) [inline public_html/feed.json]: Since the feed wasn't actually modified, it's nice to revert these ones.

**Review by guybedford — COMMENTED** [review]
Thanks for working on this and figuring out this workflow. Looks great.

It looks like this removes the `docs/generator/stable` build entirely, which I think is needed for the version dropdown?

**Comment by yowainwright** [comment]
> Thanks for working on this and figuring out this workflow. Looks great.
> 
> It looks like this removes the `docs/generator/stable` build entirely, which I think is needed for the version dropdown?

Unintentional; will fix. 👍

**Comment by yowainwright** [comment]
@guybedford updated. 👍

**Pull request merged**